crates/burn-import/src/burn/scope.rs

```diff
@@ -19,20 +19,14 @@ struct TensorVariable {
 impl Scope {
     /// Declare a new tensor variable.
     pub fn tensor_register_variable(&mut self, tensor: &TensorType, node_position: usize) {
-        if let Some(variables) = self.variables.get_mut(&tensor.name) {
-            for variable in variables.iter_mut() {
-                if variable.node_position == node_position {
-                    variable.references += 1;
-                    return;
-                }
+        if let Some(variable) = self.variables.get_mut(&tensor.name) {
+            if variable.node_position == node_position {
+                variable.references += 1;
+                return;
```

Comment:
The return here is not required (as flagged by clippy)

```rust
[clippy] reported by reviewdog 🐶
error: unneeded `return` statement
  --> crates/burn-import/src/burn/scope.rs:24:42
   |
24 |                   variable.references += 1;
   |  __________________________________________^
25 | |                 return;
   | |______________________^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_return
   = note: `-D clippy::needless-return` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::needless_return)]`
help: remove `return`
```

Reply:
sorry, completely missed this review

Reply:
No worries at all 🙂